### DISCUSSION TITLE
Retry_policy on define_asset_job with delay

### DISCUSSION THREAD

**Discussion by vvb610** [dagster-io/dagster#18759] [Q&A] [answered]
Hi, similar to the `asset` [retry_policy](https://docs.dagster.io/_apidocs/ops#dagster.RetryPolicy), is there a way to set up a retry policy in the [`define_asset_job`](https://docs.dagster.io/_apidocs/assets#dagster.define_asset_job) with a delay for the retry? I know that there is a tags parameter that allows you to specify the maximum number of retries (ie., `dagster/max_retries`) but there is no way to set the delay, which is also important in my use case. I found this related discussion [here](https://discuss.dagster.io/t/8200837/hi-how-can-i-set-op-retry-policy-on-define-asset-job-tags-da) but with no answer. Thank you!

**Comment by sryza** [comment, marked as answer]
There is not currently a way to do this, but here's the relevant tracking issue: https://github.com/dagster-io/dagster/issues/13427

**Reply by vvb610** [reply]
Thank you! Sounds good, will track the issue.
